libs/agno/agno/agent/agent.py

```diff
@@ -592,6 +592,13 @@ def set_default_model(self) -> None:
             log_info("Setting default model to OpenAI Chat")
             self.model = OpenAIChat(id="gpt-4o")
 
+    def set_memory(self) -> None:
+        if self.memory is None:
+            self.memory = Memory()
```

Comment:
Why would this still be needed? Before this was the only source of runs, so it was needed, but now I don't think it is needed by default?